applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -39,7 +39,7 @@
 else:
     AUDIO_KIT_SERIAL_NUMBERS_JSON = Path(
         os.getenv("AUDIO_KIT_SERIAL_NUMBERS_JSON"))
-TARGET_BOARD_NRF5340_AUDIO_DK_APP_NAME = "nrf5340_audio_dk_nrf5340_cpuapp"
+TARGET_BOARD_NRF5340_AUDIO_DK_APP_NAME = "nrf5340_audio_dk/nrf5340/cpuapp"
```

Comment:
Seems this is a big change, would this mean we also need to update the doc for updating the DK name when using west build?
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/nrf5340_audio/doc/building.html#building-the-application